docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -77,7 +77,24 @@ To specify more arguments, use the following syntax::
             body: "{{ mail_body }}"
           run_once: True
 
-The `ansible_host` variable reflects the host a task is delegated to.
+.. note::
+    - The `ansible_host` variable and other connection variables, if present, will reflect information of the host a task is delegated to, not the inventory_hostname.
+
+.. _delegate_parallel:
+
+Parallelism
+-----------
+By default Ansible tasks are executed in parallel. Delegating a task does not change this and does not handle concurrency issues (multiple forks writing to same file).
+Most commonly users are affected by this when updating a single file on a single delegated to host for all hosts (via copy/template/lineinfile/etc), they will still operate in parallel forks (default 5) and overwrite each other.
+
+This can be handled in several ways::
+
+    - name: "handle concurrency via a loop on the hosts with `run_once: true`"
+      lineinfile: "<options here>"
+      run_once: true
+      loop: '{{ ansible_play_hosts_all }}'
+
+By using an intermediate play with  `serial: 1` or using  `throttle: 1` at task level.
```

Comment:
Another example would be good here, and/or link to https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html#setting-the-batch-size-with-serial and https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html#restricting-execution-with-throttle.